server/services/additionalDaysAwardedBackendService.ts

```diff
@@ -191,4 +194,26 @@ export default class AdditionalDaysAwardedBackendService {
       },
     } as Adjustment
   }
+
+  public async comparisonViewModel(
+    nomsId: string,
+    activeCaseLoadId: string,
+    reqService: string,
+    token: string,
+  ): Promise<AdaComparisonModel> {
+    if (config.featureToggles.adaComparisonEnabled) {
+      const service = reqService || 'PRISON-API'
+      const prisonApiResponse = await this.adjustmentsService.getAdaAdjudicationDetails(nomsId, token)
```

Comment:
you could roll these two API calls into a Promise.all so that they both run at the same time